llvm/test/MachineVerifier/test_g_extract_subvector.mir

```diff
@@ -0,0 +1,33 @@
+# RUN: not --crash llc -o - -mtriple=arm64 -run-pass=none -verify-machineinstrs %s 2>&1 | FileCheck %s
+# REQUIRES: aarch64-registered-target
```

Comment:
Don't really need to specify aarch64, you're not referencing anything target specific

Reply:
All tests in this directory have this require. Let’s remove them together in a single follow up patch?

Reply:
I will remove this one here though!

Reply:
Updated.

Reply:
> All tests in this directory have this require. Let’s remove them together in a single follow up patch?
I will remove this one here though!

These new tests fail cleanly without expectedly crashing on builds configured for selected targets (e.g. `-DLLVM_TARGETS_TO_BUILD=ARM`) without a `LLVM_DEFAULT_TARGET_TRIPLE`. 
```
bin/llc: error: unable to get target for '', see --version and --triple.
```
I think that is why tests in this directory have the `REQUIRES` along with a valid triple; either in command line (`-mtriple=aarch64`) or in the test file itself (`target triple = "aarch64--"`).